ecs-service/datadog.tf

```diff
@@ -0,0 +1,38 @@
+data "datadog_role" "admin_role" {
+  filter = "Admin"
+}
+
+locals {
+  message = "{{#is_warning}}@slack-platform-warnings{{/is_warning}} {{#is_warning_recovery}}@slack-platform-warnings{{/is_warning_recovery}} {{#is_alert}}@opsgenie @slack-alerts{{/is_alert}} {{#is_alert_recovery}}@opsgenie @slack-alerts{{/is_alert_recovery}}"
```

Comment:
Warning na ty cpu/memory monitory se pošle jen do Slack channelu `#platform-warnings` (přejmenoval jsem ten můj testovací `#dev-platform-alerts`) a alert do Opsgenie + `#alerts`.

Reply:
Budem to asi muset trochu tunit, aby toho moc nechodilo do alertu 🤔 Premyslim, jestli to radeji nemit vsechno v platform-warnings, at to nevidi nyc :D

Reply:
Okej okej 🤔 jenom si říkám aby to nezapadlo v těch zprávách o restartech tasků. Ale nad těma teda přemýšlím jestli o nich vůbec potřebujeme vědět když máme ten opsgenie alert na tři restarty po sobě.

Reply:
ok: https://github.com/FigurePOS/terraform-modules/pull/26/commits/e75b27b9a2346fc2988bbd99e6d5bfcfb0b6aedc